src/core/linker.cpp

```diff
@@ -2,6 +2,8 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #include <Zydis/Zydis.h>
+#include <common/config.h>
+#include <common/path_util.h>
```

Comment:
```suggestion
#include "common/config.h"
#include "common/path_util.h"
```
To ensure its consistent with all the other includes